Content.Server/Tools/ToolSystem.Welder.cs

```diff
@@ -136,6 +146,17 @@ private void OnMeleeHit(EntityUid uid, WelderComponent component, MeleeHitEvent
             ItemComponent? item = null,
             PointLightComponent? light = null,
             AppearanceComponent? appearance = null)
+        {
+            bool wasLogged = false;
+            return TryTurnWelderOff(uid, user, ref wasLogged, welder, item, light, appearance);
+        }
+
+        public bool TryTurnWelderOff(EntityUid uid, EntityUid? user,
+            ref bool wasLogged,
```

Comment:
You dont need the argument `ref bool wasLogged`, it is always equal to what the method returns: false when the resolve fails and true otherwise. Remove it and use what the method returns for args.WasLogged in line 236.